src/wp-includes/block-editor.php

```diff
@@ -408,6 +408,12 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 			$block_classes['css'] = $actual_css;
 			$global_styles[]      = $block_classes;
 		}
+		// Add the custom CSS as separate style sheet so any invalid CSS entered by users does not break other global styles.
```

Comment:
The comment is too long on one line.
```suggestion
		/*
		 * Add the custom CSS as separate style sheet so any invalid CSS entered
		 * by users does not break other global styles.
		 */
```
Helps to improve readability.

Reply:
Two things:
1.  I think it needs an "a" before the word "separate".
2. Also, isn't "style sheet" one word, e.g. stylesheet?

Reply:
I can take care of committing this and address this feedback prior to the commit.